README.md

```diff
@@ -0,0 +1,16 @@
+# CompilerWebApp
+
+CompilerWebApp is a web server that allows users to study their code online without any burden of environment setup. 
+
+## Major Features
+* Customers can input, edit, view the code on the main product’s page as well as choose toolchains & compilations options
+* Customers can submit their code and receive the results of the compilation
+* Administrators can deploy the backend using Docker
```

Comment:
Have you tried this, not sure this is tested well enough to be mentioned at this stage?

Reply:
Same for some points below

Reply:
Thanks a lot - you are quite right. Readme was changed and not implemented features were removed from it.